data/map planets.txt

```diff
@@ -149,6 +154,10 @@ planet Allhome
 	landscape land/dmottl0
 	description `This Earth-like planet has been settled both by human beings and by the Hai, the species that controls this region of space. The human settlements are relatively small, mostly farming communities founded by people who came here to escape the chaos and uncertainty of human space. More substantial urban settlements exist, particularly near the spaceport, but from the air they appear to exist within strict boundaries.`
 	spaceport `In the spaceport, a handful of human merchants and colonists wander about amid throngs of Hai - vaguely squirrel-like aliens who are somewhat shorter than the average human being, but make up for it with an abundance of energy. You feel a little bit like an adult wading through a sea of hyperactive children as you try to locate the commodity exchange and the other services offered here.`
+	spaceport `	Despite the distance from Republic space, some humans have set up decorations in celebration of Remembrance Day. A handful of the more curious Hai approach the humans, asking them what the fuss is all about.`
```

Comment:
Hai approaching the humans implies that a mission is about to be offered, and seems like an event instead of a description. Maybe:
```suggestion
	spaceport `	Despite the distance from Republic space, some humans have set up decorations in celebration of Remembrance Day. A handful of the more curious Hai mingle with the humans, asking them what the fuss is all about.`
```